app/helpers/check_answers_helper.rb

```diff
@@ -1,10 +1,21 @@
 module CheckAnswersHelper
+  SKIPPABLE_QUESTIONS = %w(
+    are_you_a_manufacturer
+    medical_equipment_type
+    product_details
+    hotel_rooms_number
+    transport_type
+    offer_space_type
+    offer_care_qualifications
+  ).freeze
+
   def items
     questions.map { |question|
       # We have answers as strings and hashes. The hashes need a little more
       # work to make them readable.
 
       next if question.eql?("medical_equipment_type")
```

Comment:
Do we need to keep this given that it's now in skippable questions? Am I reading it right that under the old system this just wouldn't have shown up in the answers?

Reply:
You're right! I'll go and update this.

Reply:
@JonathanHallam updated here: https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/pull/194/commits/fcf6cd5123c842650c59664bd9ca7d1795cef095